docs/data/base/getting-started/usage/usage.md

```diff
@@ -20,28 +20,32 @@ function App() {
 ```
 
 You can play around with this code in the interactive Code Sandbox demo below.
-Try importing an `InputUnstyled` component and adding it to the `<div>`:
+Try importing an [`InputUnstyled`](/base/react-input/) component and adding it to the `<div>`:
 
 {{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
 
 ## Shared props
 
 Base components are self-supporting and fully functional in isolation.
 
-Each component has its own unique API, but all components accept the following shared props:
+Each component has its own unique API, but all _non-utility_ components accept the following shared props:
 
 ### components
 
-The `components` prop is an object that lets you override any interior subcomponents ("slots") of the base component itself.
+The `components` prop is an object that lets you override any interior subcomponents—known as **slots**—of the base component itself.
 
 :::info
-Each component contains a `Root` slot, and many complex components (such as [`BadgeUnstyled`](/base/react-badge/)) are composed of additional subcomponents that can be customized.
-:::
+Each component contains a root slot, and other appropriate slots based on the nature of the component.
+For example, the `BadgeUnstyled` contains two slots:
+
+- `root`: the container element that wraps the children.
+- `badge`: the element that appears at the given position.
```

Comment:
"the element that appears at the given position" sounds pretty vague. `components.Badge` represents a component that is the badge itself (i.e., a dot with a number inside).

Reply:
That is very strange wording, good catch.